tests/integration/security/rbac_v1beta1_test.go

```diff
@@ -744,11 +752,10 @@ func TestV1beta1_TCP(t *testing.T) {
 				// The policy on workload b denies request with path "/data" to port 8090:
 				// - request to port http-8090 should be denied because both path and port are matched.
 				// - request to port http-8091 should be allowed because the port is not matched.
-				// - request to port tcp should be denied because a default deny-all policy should
-				//   be generated when HTTP field (i.e. path) is used on TCP port.
+				// - request to port tcp should be denied because the port is not matched.
 				newTestCase(a, b, "http-8090", false),
 				newTestCase(a, b, "http-8091", true),
-				newTestCase(a, b, "tcp", false),
+				newTestCase(a, b, "tcp", true),
```

Comment:
Does this mean request to port tcp is allowed? It doesn't match with your comment.

Reply:
Oh, the comment is wrong. Here the request should be allowed because the deny policy specifies port 8090 explicitly which means it should have no effect on other port (i.e. tcp)

Reply:
So this is changed compare to the previous behavior, right?